pulsar-broker/src/main/java/com/yahoo/pulsar/broker/loadbalance/impl/LinuxBrokerHostUsageImpl.java

```diff
@@ -160,19 +155,10 @@ private ResourceUsage getMemUsage() {
 
     private boolean isPhysicalNic(Path path) {
         try {
-            path = Files.isSymbolicLink(path) ? Files.readSymbolicLink(path) : path;
-            if (!path.toString().contains("/virtual/")) {
-                try {
-                    Files.readAllBytes(path.resolve("speed"));
-                    return true;
-                } catch (Exception e) {
-                    // wireless nics don't report speed, ignore them.
-                    return false;
-                }
-            }
-            return false;
-        } catch (IOException e) {
-            LOG.error("Failed to read link target for NIC " + path, e);
+            Files.readAllBytes(path.resolve("speed"));
+            return true;
+        } catch (Exception e) {
+            // wireless nics don't report speed, ignore them.
```

Comment:
We decided that, since virtual nics will not be able to resolve speed (?), we may as well let catch block deal with it. That way, we do not need to resolve symbolic links or get absolute paths.